packages/main/src/modules/cli.ts

```diff
@@ -52,6 +53,7 @@ export async function deploy({ path, target, targetContent }: DeployOptions) {
     ],
     cwd: path,
     env: await getEnv(path),
+    workspace: path,
```

Comment:
We would need to make sure the `node_modules` folder exists, and if not, do an `npm install`, before running the CLI